backend/src/schema/resolvers/reports.spec.js

```diff
@@ -1,37 +1,73 @@
 import { GraphQLClient } from 'graphql-request'
 import Factory from '../../seed/factories'
-import { host, login } from '../../jest/helpers'
-import { neode } from '../../bootstrap/neo4j'
+import { host, login, gql } from '../../jest/helpers'
+import { getDriver, neode } from '../../bootstrap/neo4j'
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../.././server'
 
 const factory = Factory()
 const instance = neode()
+const driver = getDriver()
 
-describe('report', () => {
+describe('report mutation', () => {
   let reportMutation
   let headers
-  let returnedObject
+  let client
   let variables
   let createPostVariables
   let user
   const categoryIds = ['cat9']
 
+  const action = () => {
+    reportMutation = gql`
+      mutation($resourceId: ID!, $reasonCategory: ReasonCategory!, $reasonDescription: String!) {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T19:26:23Z" title="Monday, October 14th 2019, 9:26:23 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,7 +20,7 @@ describe('report mutation', () => {
 
   const action = () => {
     reportMutation = gql`
-      mutation($resourceId: ID!, $reasonCategory: String!, $reasonDescription: String!) {
+      mutation($resourceId: ID!, $reasonCategory: ReasonCategory!, $reasonDescription: String!) {
```


Thanks that you have seen this and the related ones. 👍🏼 I was obviously in a hurry …
Sorry for that!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T19:29:04Z" title="Monday, October 14th 2019, 9:29:04 pm +02:00">Oct 14, 2019</time>_

---


no need to apologize, we got each others' backs